src/main/java/io/github/thebusybiscuit/mobcapturer/mobs/ZombifiedPiglinAdapter.java

```diff
@@ -1,27 +1,43 @@
 package io.github.thebusybiscuit.mobcapturer.mobs;
 
+import com.google.gson.JsonObject;
+import io.github.thebusybiscuit.slimefun4.utils.ChatUtils;
+import org.bukkit.ChatColor;
 import org.bukkit.entity.PigZombie;
 
-import com.google.gson.JsonObject;
+import java.util.List;
 
 public class ZombifiedPiglinAdapter extends AbstractHumanoidAdapter<PigZombie> {
 
     public ZombifiedPiglinAdapter() {
         super(PigZombie.class);
     }
 
+    @Override
+    public List<String> getLore(JsonObject json) {
+        List<String> lore = super.getLore(json);
+
+        lore.add(ChatColor.GRAY + "Anger: " + ChatColor.WHITE + json.get("anger").getAsString());
+        lore.add(ChatColor.GRAY + "Baby: " + ChatColor.WHITE + json.get("baby").getAsBoolean());
+
+        return lore;
+    }
+
     @Override
     public void apply(PigZombie entity, JsonObject json) {
         super.apply(entity, json);
 
         entity.setAnger(json.get("anger").getAsInt());
+        entity.setAge(json.get("age").getAsInt());
```

Comment:
you aren't applying baby here, is this done in super?

Reply:
That's good, I'm applying age. setBaby just sets the age to baby. However, having an age in lore would be quite confusing for regular player, so I'm just putting into the lore whether it's a baby, which is the part that the player can actually see.